src/library-authoring/data/api.ts

```diff
@@ -111,6 +111,10 @@ export const getLibraryContainersApiUrl = (libraryId: string) => `${getApiBaseUr
  * Get the URL for the container detail api.
  */
 export const getLibraryContainerApiUrl = (containerId: string) => `${getApiBaseUrl()}/api/libraries/v2/containers/${containerId}/`;
+/**
+ * Get the URL for a single container children api.
+ */
+export const getLibraryContainerChildrenApiUrl = (containerId: string) => `${getApiBaseUrl()}/api/libraries/v2/containers/${containerId}/children/`;
```

Comment:
I think this is ok for now, but we should put this info in our index. Each of these calls took almost 2 sec on my stack.

Since we don't have the children's info on the index yet (and don't plan to have it soon), we could store an array with the first `N` types on the Unit document, like you did here in the frontend.
What do you think?

Reply:
I agree we should put the list of child types in the index (can come in a separate PR). But I wonder why each of these calls takes 2 seconds? That seems very slow and it should be super fast considering our unit code is very new and supposedly lightweight.

Reply:
I think it is because I had a lot of concurrent requests?
![image](https://github.com/user-attachments/assets/e079e96a-d578-4a1e-8c5f-7d66a42987ad)

Also, we are iterating through the records here:
https://github.com/openedx/openedx-learning/blob/a79b579fd3f91c763d193c2e6841fb7f9f4796c7/openedx_learning/apps/authoring/units/api.py#L245-L267

Reply:
Created a followup task to fix this: https://github.com/openedx/frontend-app-authoring/issues/1778

Reply:
Perhaps we are missing a select_related then. Iterating through the records should still be fast, as long as there are no database queries within the loop. Perhaps we need to select_related on `entry.entity_version.componentversion` outside the loop before accessing it inside the loop.